### ISSUE TITLE
Stick war 2  is laggy and slow and background is not showing

### ISSUE THREAD

**Issue by DEVA5581** [ruffle-rs/ruffle#11009]
### Describe the bug

I
![Screenshot 2023-05-11 19 53 11](https://github.com/ruffle-rs/ruffle/assets/115197604/e40468e4-7fea-4ead-a03d-94a736e8f75a)
 already wrote that on the title anyway how to disable right-click in the browser demo?

### Expected behavior

the background is rendered and at least less lag

### Affected platform

Online demo

### Operating system

chrome OS

### Browser

google chrome

**Comment by DEVA5581** [comment]
[3914.zip](https://github.com/ruffle-rs/ruffle/files/11452418/3914.zip)

**Comment by n0samu** [comment]
Right-click controls are unimplemented (see #10288), so the game isn't playable yet. I don't know why the background isn't showing up though.

**Comment by MartySVK** [comment]
looks like the game is working now
